.gitignore

```diff
@@ -1,6 +1,8 @@
 build/
+out/
```

Comment:
Because @jtanx asked for my opinion:

I continue to disagree with the change to the Git ignore. There is no good reason for us to have two build folders. Configure your IDE. We shouldn't be expected to bend over backwards to support the defaults of every IDE. If there's a configuration file we can add to tell VS to use our build folder, that's acceptable too.

Reply:
Ok.